examples/v1beta1/hp-tuning/random.yaml

```diff
@@ -62,4 +62,8 @@ spec:
                   - "--lr=${trialParameters.learningRate}"
                   - "--num-layers=${trialParameters.numberLayers}"
                   - "--optimizer=${trialParameters.optimizer}"
+                resources:
+                  limits:
+                    memory: "1Gi"
+                    cpu: "0.5"
```

Comment:
We just discussed this with @johnugeorge during the AutoML WG sync. Let's keep this resources since we have `ResourceQuota` in our Conformance profile.